modules/exploits/unix/http/splunk_xslt_authenticated_rce.rb

```diff
@@ -0,0 +1,352 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => '(Authenticated) RCE in Splunk (9.0.x to 9.0.7, 9.1.x to 9.1.2)',
+        'Description' => %q{
+          This exploit module targets a Remote Code Execution vulnerability in Splunk Enterprise.
```

Comment:
I would expand this about how you upload an xslt file w/ a payload in it, and then run a processor against it to execute the payload